src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/BitHelper.cs

```diff
@@ -1,6 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#if NETCOREAPP
```

Comment:
Why this ifdef rather than conditionally including/excluding the file in the csproj?

Reply:
In msbuild you need to write something more involved like `Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)' or '$(TargetFramework)' == 'netcoreapp3.0'"` which is probably more prone breaks as supported targets fluctuate.